_includes/configuration/toc_header.md

```diff
@@ -0,0 +1,19 @@
+### `toc_header`
+
+Table of contents has a header and by default it will be marked up with `H2` tag. With `toc_header` option you can change it to some other tag.
```

Comment:
```suggestion
Table of contents has a header and by default it will be marked up with `H2` tag. With the `toc_header` option you can change it to some other tag.
```